avm/res/network/private-endpoint/README.md

```diff
@@ -25,7 +25,7 @@ The following section provides usage examples for the module, which were used to
 
 >**Note**: Each example lists all the required parameters first, followed by the rest - each in alphabetical order.
 
->**Note**: To reference the module, please use the following syntax `br/public:avm-res-network-privateendpoint:1.0.0`.
+>**Note**: To reference the module, please use the following syntax `br/public:avm/res/network/private-endpoint:0.2.0`.
```

Comment:
Not sure about this. This would mean regenerating readmes every time version.json is updated. Also, I guess it would only be possible to update major/minor.

Reply:
It's actually the version calculated via our own versioning tool. 
I tried to fight this, but we got some very strong-minded feedback from the community. I just added it now as I was changing that exact line anyways.